econml/inference/_inference.py

```diff
@@ -1310,7 +1310,7 @@ def percentile_point(self, *, alpha=.1):
         upper_percentile_point = np.percentile(self.pred, (1 - alpha / 2) * 100, axis=0)
         return lower_percentile_point, upper_percentile_point
 
-    def conf_int_point(self, *, alpha=.1, tol=.001):
+    def conf_int_point(self, *, alpha=None, tol=.001):
```

Comment:
As @heimengqi pointed out, it would be great to also address `tol` while you're at it.